### ISSUE TITLE
The type or namespace name 'Extensions' does not exist in the namespace

### ISSUE THREAD

**Issue by acrigney** [unoplatform/uno.extensions#859]
### Current behavior

Guys I have Uno templates and unocheck installed with no errors, using VS2022 Version 17.3.6.
I created a project using the instructions here
https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/ExtensionsOverviewAndGettingStarted.html
But when I build the project I get that the Extensions are not found but I see they are all referenced in the project. 
This is a code generation error?
I deleted the bin and obj folders and tried several times.
[PCSProducts.Uno.zip](https://github.com/unoplatform/uno/files/9848006/PCSProducts.Uno.zip)

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0234	The type or namespace name 'Extensions' does not exist in the namespace 'PCSProducts.Uno' (are you missing an assembly reference?)	PCSProducts.Uno	C:\Projects\PCSProducts.Uno\PCSProducts.Uno\PCSProducts.Uno\Uno.Extensions.Reactive.Generator\Uno.Extensions.Reactive.Generator.FeedsGenerator\PCSProducts.Uno.Presentation.MainViewModel.g.cs	28	Active

### Expected behavior

Project should build and run fine.

**Comment by dr1rrb** [comment]
![image](https://user-images.githubusercontent.com/8635919/197616996-4ec2049e-abac-4d79-afed-fa13205e76f8.png)
 This is due to the mapped ctor parameters from the VM to the BindableVM, we should add more `global::`

**Comment by acrigney** [comment]
Awesome mate! After more than 20 years in .net I have never used global usings before, so this is as the generated code doesn't generate the using statements?

**Comment by dr1rrb** [comment]
No, "global using" is a new feature of C#10 that allows you to define at assembly level some "using", e.g. `global using System.Linq;`. Once declared you can use that namespace in all your classes without having to import it again and again. There is also a generator that will generate some _global usings_ based on your referenced packages (cf. https://dotnetcoretutorials.com/2021/08/19/global-using-statements-in-c10/)

Here the issue is that we have a namespace conflict with the application namespace: As you have defined a namespace `PCSProducts.**Uno**`, the compiler tries to resolve the `Uno.Extensions.[...]` namespace relative to this NS (i.e. `PSProducts.Uno.Extensions`)  instead of the considering it as an absolute namespace. To resolve this, we have to add the namepsace alias qualifier (https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/namespace-alias-qualifier). Basically it's just like an Uri where we add the `https://` to make it absolute.
